### PR TITLE
leftover dots and dashes

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 1]
```diff
@@ -96,14 +96,14 @@ To be honest, I couldn't stand that smile anymore.
 "Why do you have lines all over your body?
 And there are so many cracks in this room as well, it looks like it's about to fall apart."
 <Page27>
-Although his smile vanished for a brief moment, he quickly returned to his smiling face and stood up from his chair.
+Although his smile vanished for a brief moment, he quickly returned to his smiling face and stood up from his chair.//'he quickly returned to his smiling face' sounds weird, needing rephrasing?
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Common/00_00.txt:99]: Although his smile vanished for a brief moment, it quickly returned and he stood up from his chair.

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 2]
```diff
@@ -96,14 +96,14 @@ To be honest, I couldn't stand that smile anymore.
...
 The doctor clattered across the floor and vanished behind a curtain.
 <Page28>
 "...It seems there was some brain damage after all.
 Contact Doctor Ashiya in the neurosurgery department. I also suspect some damage to the optic nerves. Send him in for an eye examination this afternoon."
-The doctor whispered to the nurse in a cold voice, as if he was an entirely different person.
+The doctor whispered to the nurse in a cold voice, as if he were an entirely different person.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Common/00_00.txt:104]: I like
> The doctor whispered to the nurse in the cold voice of an entirely different person.

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 3]
```diff
@@ -228,17 +228,17 @@ But before that.
...
 <Page62>
 The woman wryly pointed at me.
-......I was a little pissed.
+...I was a little pissed.
 I was the fourth tallest in my class, so I didn't think I was that short.
 <Page63>
 "Who's going to get sent flying?"
 "Don't be ridiculous, that should be obvious.
-We were the only ones here, so if you counted me out then who's left?"
+We are the only ones here, so if you counted me out, then who's left?"//Shinyyyjoke suggestion
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Common/00_00.txt:241]: this line is addressed in Orange's PR.

**Review by Avuxo — APPROVED** [review]
approved pending a few changes. Merge this before Orange's since the conflicts will be more manageable that way.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt [hunk 4]
```diff
@@ -1,12 +1,12 @@
 <Page1>
 The rail line above ground switches to an underground line without a hint of warning.
-The train, spewing artificial light, moves through the darkness as if it were diving into it.
+The train, emitting artificial light, moves through the darkness as if it were diving into it.//Shinyyyjoke suggestion
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt:3]: I kinda disagree with this change, spewing is far more evocative imo

File: script/1_0_2current_en_script/Common/00_00.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Common/00_00.txt:104]: Kinda prefer it as is tbh

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 5]
```diff
@@ -284,16 +284,16 @@ I was glad. For the first time since I woke up from the accident, I felt human a
...
 The woman would get upset if I called her Aoko. She said she didn't like her name.
-After much pondering, I decided to call her "Sensei" because she seemed to be an important person.
+After much pondering, I decided to call her 'Sensei' because she seemed to be an important person.
 <Page79>
 Sensei listened to me seriously no matter what I said and solved my problems with just a few words.
 <Page80>
 After the accident, I had become uncomfortable talking to people, but thanks to her, I was able to gradually return to my normal self.
 <Page81>
-Only when I was in this field did my anxiety disappear because I knew that I could be my normal self here.
+Only after I was in this field did my anxiety disappear because I realized I could be myself.//Shinyyyjoke suggestion
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Common/00_00.txt:293]: Is the jp text "_while_ I was in this field I did not feel anxiety", as in he is non anxious when not in the field, or is it that he is _never_ anxious after being there once?

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 6]
```diff
@@ -306,12 +306,12 @@ I traced the line and cut out half of the tree from the middle, like cutting a p
 <Page86>
 "――――――"
 Sensei was surprised.
-I got really good at it, so I thrust my knife into the base of the tree, in a horizontal line.
+I got really good at it, so in a horizontal line, I thrust my knife into the base of the tree.//Shinyyyjoke suggestion
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Common/00_00.txt:309]: Reads slightly less naturally to me

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt (hunk 4 above)
Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt:3]: まき散らし

agreed, spewing is more apt

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 7]
```diff
@@ -36,11 +36,11 @@ the sight of my sister being scolded by that man,
 and even the sight of the doctor frolicking like a child,
 all of which, I remembered as though it were a dream.
 <Page11>
-"Huh? You're not actually awake, are you!?"
+"Eh? You're not actually awake, are you...?!"
```
Comment (yk1512) [inline script/1_0_2current_en_script/Common/00_00.txt:39]: Eh is the direct japanese onomatopoeia but I think huh is better as a translation.

Added ellipses is ok.

File: script/1_0_2current_en_script/Common/00_00.txt (hunk 2 above)
Comment (yk1512) [reply script/1_0_2current_en_script/Common/00_00.txt:104]: I agree with ross

File: script/1_0_2current_en_script/Common/00_00.txt (hunk 5 above)
Comment (yk1512) [reply script/1_0_2current_en_script/Common/00_00.txt:293]: > この野原にいる時だけは

only while I was in this field. 

I think original is better

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 8]
```diff
@@ -36,11 +36,11 @@ the sight of my sister being scolded by that man,
...
 I heard an unfamiliar girl's voice.
 I didn't even have the strength to answer her.
 <Page12>
-Panicking, the woman darted out of the room calling for a doctor.
+Panicking, the girl darted out of the room calling for a doctor.//女の子
```
Comment (Volvo-hackangel) [inline script/1_0_2current_en_script/Common/00_00.txt:43]: is this change alright?

Comment (rschlaikjer) [reply script/1_0_2current_en_script/Common/00_00.txt:43]: If the JP is 女の子 then yeah. Plus, it makes it consistent with L40

**Comment by rschlaikjer** [comment]
Did we not want the remaining changes here? They seem OK?

**Pull request closed without merging**